app/routes.py

```diff
@@ -1,2 +1,237 @@
-from flask import Blueprint
+from flask.wrappers import Response
+from sqlalchemy.orm import query
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request
+from datetime import datetime
+import os
+import requests
+from app.models.goal import Goal
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["GET", "POST"])
+def handle_tasks():
+    if request.method == "POST":
+        
+        request_body = request.get_json()
+        
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return {
+                "details": "Invalid data"
+            },400
+        new_task = Task(title=request_body["title"],
+                        description=request_body["description"],
+                        completed_at=request_body["completed_at"])
+        is_complete = new_task.completed_at is not None
+
+        db.session.add(new_task)
+        db.session.commit()
+
+        return {'task':
+            {"id": new_task.task_id,
+            "title": new_task.title,
+            "description": new_task.description,
+                
+            "is_complete": is_complete
+            }},201
```

Comment:
I recommend using existing helper functions whenever possible.

```suggestion
        return {'task': new_task.to_dict()},201
```